examples/temporal/README.md

```diff
@@ -0,0 +1,97 @@
+# Running SkyPilot Tasks in Temporal Workflows
+
+This example demonstrates how to launch SkyPilot tasks and manage them in a Temporal workflow. 
+
+<p align="center">
+  <img src="https://i.imgur.com/rxlO2pJ.png" width="800">
+</p>
+
+## Prerequisites
+
+- [SkyPilot](https://docs.skypilot.co/en/latest/getting-started/installation.html) installed on temporal workers
+- [Remote SkyPilot API server](https://docs.skypilot.co/en/latest/reference/api-server/api-server.html)
+
+Using the SkyPilot API Server eliminates the need for managing SkyPilot state across workers and allows credentials to be stored securely in the API server.
+
+To use the API server, set the `SKYPILOT_API_SERVER_ENDPOINT` environment variable:
+
+```bash
+export SKYPILOT_API_SERVER_ENDPOINT=https://your-api-server-endpoint
+```
+
+You can also set this in a `.env` file in the same directory as the scripts.
+
+## Defining the Tasks
+
+We will define the following tasks to mock a training workflow:
+1. **`data_preprocessing.yaml`**: Generates data and writes it to a bucket.
+2. **`train.yaml`**: Trains a model using the data in the bucket.
+3. **`eval.yaml`**: Evaluates the model and writes the evaluation results to the bucket.
+
+These tasks are defined in the [mock-train-workflow](https://github.com/skypilot-org/mock-train-workflow/tree/clientserver_example) repository. The repository is cloned during the workflow to execute the tasks.
```

Comment:
Is it easy to inline these here? Much easier to click around & they will be inlined into docs.